homework/HW_05-01/src/nyc/c4q/m47bell/PlayingCards.java

```diff
@@ -0,0 +1,20 @@
+package nyc.c4q.m47bell;
+/**
+ * Created by c4q-marbella on 5/1/15.
+ * Access Code 2-1
+ * Marbella Vidals
+ */
+public class PlayingCards extends Game
+{
+    public int cardNum = 52;
```

Comment:
Good idea creating a variable for number of cards. Maybe change this to final or private so that it can't be changed by someone else? Since 52 should never change.